db/connect.js

```diff
@@ -0,0 +1,25 @@
+const mongoose = require("mongoose");
+
+const database = "soundcloud";
+
+mongoose.connect(`mongodb://localhost/${database}`, {
+  useNewUrlParser: true,
+  useUnifiedTopology: true,
+  useCreateIndex: true,
+  useFindAndModify: false,
+});
+
+const db = mongoose.connection;
+
+db.on("error", console.error.bind(console, "connection error:"));
+
+db.once("open", function () {
+  console.log(`mongodb connected to db "${database}"!`);
+});
+
```

Comment:
This is an interesting way to connect to the database., haven't tried this way before.